architecture/additional_concepts/storage.adoc

```diff
@@ -32,12 +40,12 @@ High-availability of storage in the infrastructure is left to the underlying
 storage provider.
 ====
 
-A `*PersistentVolumeClaim*` (PVC) object represents a request for storage by a
-user. It is similar to a pod in that pods consume node resources and PVCs
-consume PV resources. For example, pods can request specific levels of resources
-(e.g., CPU and memory), while PVCs can request specific
-link:#pv-capacity[storage capacity] and link:#pv-access-modes[access modes]
-(e.g, they can be mounted once read/write or many times read-only).
+PVCs are defined by a `*PersistentVolumeClaim*` API object, which represents a
+request for storage by a developer. It is similar to a pod in that pods consume
+node resources and PVCs consume PV resources. For example, pods can request
+specific levels of resources (e.g., CPU and memory), while PVCs can request
+specific link:#pv-capacity[storage capacity] and link:#pv-access-modes[access
+modes] (e.g, they can be mounted once read/write or many times read-only).
```

Comment:
@pmorie is the sentence above correct -- "e.g, they can be mounted once read/write or many times read-only"?
My understanding is that `accessModes` has nothing at all to do with actual access. Instead `accessModes` are merely labels to help match a PV and a PVC. Assuming that is true, then do these labels really impact how a volume is mounted?